lua/tardis/libraries/sh_messages.lua

```diff
@@ -14,9 +14,9 @@ function TARDIS:Debug(text, ply)
 	if TARDIS:IsDebugOn() then
 		local fulltext = "[TARDIS Debug] "..text
 		if CLIENT and LocalPlayer() then
-			LocalPlayer():ChatPrint(fulltext)
+            TARDIS:Message(LocalPlayer(), fulltext)
```

Comment:
This should be another exception

Reply:
+, debug should stay as is, haven't noticed that - thanks